stdlib/public/Observation/Sources/Observation/Observable.swift

```diff
@@ -10,11 +10,36 @@
 //===----------------------------------------------------------------------===//
 
 
+/// A type that emits notifications to observers when underlying data changes.
+///
+/// Conforming to this protocol signals to other APIs that the type supports
+/// observation. However, applying the `Observable` protocol by itself to a
+/// type doesn't add observation functionality to the type. Instead, always use
+/// the ``Observation/Observable-swift.macro`` macro when adding observation
+/// support to a type.
 @available(SwiftStdlib 5.9, *)
 @_marker public protocol Observable { }
 
 #if $Macros && hasAttribute(attached)
 
+/// Defines and implements conformance of the Observable protocol.
+///
+/// This macro adds observation support to a custom type and conforms the type
+/// to the ``Observation/Observable-swift.protocol`` protocol. For example, the
+/// following code apply the `Observable` macro to the type `Car` making it
+/// observable:
+///
+/// ```swift
+/// @Observable class Car {
```

Comment:
The usual style for attributes is that they go on the same line as single-line declarations, and on their own line before multi-line declarations.

```suggestion
/// @Observable
class Car {
```